doc/api.rst

```diff
@@ -33,6 +33,22 @@ ListeningSystem class
    :inherited-members:
 
 
+SendingSystem class
+~~~~~~~~~~~~~~~~~~~
```

Comment:
In the rest of the doc (section development) we used `-` to indicate a subsetion of `=`, following this order: `*`, `=`, `-`, `~`. We should also change the previous sections according to this convention.